tests/one_taskset.rs

```diff
@@ -159,32 +159,58 @@ fn test_single_taskset_empty_config_host() {
 // Load test with a single task set containing two weighted tasks setup via closure.
 // Validate weighting and statistics.
 fn test_single_taskset_closure() {
+    // @todo Move out to common.rs.
+    #[derive(Debug)]
+    struct LoadtestEndpoint<'a> {
```

Comment:
Overall it would be nice to not have that be test-only, but be usable also by library users so they don't have to re-invent the wheel.

Reply:
I mean that it would be cool to be able to specify a lost test just by creating this structure and then it creates the tasks automatically.